lib/p2p/Peer.ts

```diff
@@ -197,12 +226,19 @@ class Peer extends EventEmitter {
   /**
    * Prepares a peer for use by establishing a socket connection and beginning the handshake.
    * @param ownNodeState our node state data to send to the peer
-   * @param nodeKey our identity node key
+   * @param ownNodeKey our identity node key
    * @param expectedNodePubKey the expected nodePubKey of the node we are opening a connection with
    * @param retryConnecting whether to retry to connect upon failure
    * @returns the session init packet from beginning the handshake
    */
-  public beginOpen = async (ownNodeState: NodeState, nodeKey: NodeKey, expectedNodePubKey?: string, retryConnecting = false):
+  public beginOpen = async ({ ownNodeState, ownNodeKey, ownVersion, expectedNodePubKey, retryConnecting = false }:
```

Comment:
nit: ownVersion is missing from the typedoc

Reply:
I noticed that with destructuring, typedoc is no longer showing the comments for the parameters via `@param`. I'm going to switch to the suggestion in this comment since that seems to work: https://github.com/TypeStrong/typedoc/issues/653#issuecomment-504444480